disk-deactivate/disk-deactivate.jq

```diff
@@ -43,6 +43,10 @@ def deactivate:
       "cryptsetup luksClose \(.path)",
       "wipefs --all -f \(.path)"
     ]
+  elif .type == "swap" then
+    [
+      "swapoff \(.path)",
```

Comment:
Is the trailing comma correct? I'm getting an error during disko formatting around this line since I've updated to the latest disko release.

Reply:
Here's a PR that removed the trailing comma: #992